Sources/MongoSwift/MongoClient.swift

```diff
@@ -402,6 +405,106 @@ public class MongoClient {
         )
     }
 
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+    @available(macOS 10.15, *)
+    internal class CmdHandler: CommandEventHandler {
+        private var continuation: AsyncStream<CommandEvent>.Continuation
+        internal init(continuation: AsyncStream<CommandEvent>.Continuation) {
+            self.continuation = continuation
+        }
+
+        // Satisfies the protocol
+        internal func handleCommandEvent(_ event: CommandEvent) {
+            self.continuation.yield(event)
+        }
+
+        internal func finish() {
+            self.continuation.finish()
+        }
+    }
+
+    @available(macOS 10.15, *)
+    internal class SDAMHandler: SDAMEventHandler {
+        private var continuation: AsyncStream<SDAMEvent>.Continuation
+        internal init(continuation: AsyncStream<SDAMEvent>.Continuation) {
+            self.continuation = continuation
+        }
+
+        // Satisfies the protocol
+        internal func handleSDAMEvent(_ event: SDAMEvent) {
+            self.continuation.yield(event)
+        }
+
+        internal func finish() {
+            self.continuation.finish()
+        }
+    }
+
+    /// Provides an `AsyncSequence` API for consuming command monitoring events.
+    /// Example: printing the command events out would be written as
+    /// `for await event in client.commandEventStream() { print(event) }`.
+    /// Wrapping in a `Task { ... }` may be desired for asynchronicity.
+    /// Note that only the most recent 100 events are stored in the stream.
```

Comment:
Swift markup let's us use special keywords to highlight things. One such example is "Note". To use it, be sure to write it like this: `-  Note: <note text>`. Ditto for the SDAM one.